pkg/proxy/proxy/transparent.go

```diff
@@ -165,7 +179,7 @@ func (t TransparentProxyService) ProxyTLS(addr string) {
 	}
 }
 
-func (t *TransparentProxyService) ServeMetadata(addr string) {
+func (t *TransparentProxyService) ServeAdminEndpoint(addr string) {
```

Comment:
Hmm, if we're speaking about the KoRBE use case then I don't think we need to secure the endpoint, since the user could bypass the proxy in other ways if they broke out of the docker container. This would probably just add some overhead in maintenance on how we call the proxy.

What type of auth mechanism did you have in mind?

Reply:
Will follow up on this in the future if we need to add auth